### PR TITLE
Center map on location

### PR DESCRIPTION
**Pull request by iamdeaneyelid**
I've updated the venues map so you can give it a custom initial location. 

This can be done by adding a location and it's longitude/latitude in the options page on the CMS

Then appending _"l=[name_of_location]"_ to the /map url
i.e http://local.emeraldstreet.com/map?l=manchester

screen shot 2016-10-17 at 18 28 39

_ACF fields in the options page to enter location info_

map-es

_Switching locations_

### CODE REVIEW

**Review by jonsherrard — APPROVED** [review]
Excellent. :+1:

File: app/panels.php [hunk 1]
```diff
@@ -208,6 +208,84 @@
       'disabled' => 0,
     ),
     array (
+      'key' => "{$ns}_default_location",
+      'label' => 'Map Default Location',
+      'name' => 'map_default_location',
+      'type' => 'repeater',
+      'instructions' => '',
+      'required' => 0,
+      'conditional_logic' => 0,
```
Comment (ecoad) [inline app/panels.php:112]: Can you clean up all of these unused ACF configurations?

File: app/panels.php [hunk 2]
```diff
@@ -208,6 +208,84 @@
       'disabled' => 0,
     ),
     array (
+      'key' => "{$ns}_default_location",
+      'label' => 'Map Default Location',
```
Comment (ecoad) [inline app/panels.php]: What's a better name here.

"Default" = It isn't really default as there is a default fallback, it's more like "Centred"/"Starter"/"Initial"

"Location" to me just sounds like there's one whereas it's a repeater field, so it should be plural

so maybe `Map Initial Locations`? Pick one of them and update all other references throughout your PR

File: src/components/Map.js [hunk 3]
```diff
@@ -8,10 +8,25 @@ class Map extends Component {
...
+      locationLat: 51.507,
+      locationLon: -0.128,
+    }
     this.handleMoveEnd = this.handleMoveEnd.bind(this)
     this.handleLocationFound = this.handleLocationFound.bind(this)
   }
 
+  getDefaultLocation() {
+    let url = window.location.search.substr(1);
+    let locationName = url.match(/l=([^&]*)/);
```
Comment (ecoad) [inline src/components/Map.js:21]: 👍

**Review by ecoad — COMMENTED** [review]
Looking good. Just a couple of small changes.

**Review by thomasdigby — APPROVED** [review]
👍

**Comment by thomasdigby** [comment]
@jonsherrard @ecoad Are we happy to merge this?

**Review by jonsherrard — APPROVED** [review]
Mergable

**Comment by thomasdigby** [comment]
@jonsherrard Will this need to be merged into master?

**Comment by jonsherrard** [comment]
@thomasdigby Yes it'll need to be merged into master, and tagged, and for the tags to be pushed to Github.

`git pull origin master --tags` to get the latest tag.

**Comment by thomasdigby** [comment]
Alright, I'll take a look at this later.

**Pull request merged**